app/retail/templates/emails/grants/subscription_terminated.html

```diff
@@ -0,0 +1,172 @@
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n static %}
+
+{% block content %}
+
+<style>
+  hr {
+    width: 80%;
+    height: 2px;
+    border-radius: 25px;
+    border: none;
+    background-color: #D2D2D2;
+  }
+
+  .grantInfo {
+    padding-top: 30px;
+    padding-bottom: 1%;
+  }
+
+  .grantInfo a img {
+    border-radius: 25px;
+    margin-bottom: 10px;
+  }
+
+  .grantInfoName {
+    font-size: 20px;
+    color: #0D0764;
+  }
+
+  .grantInfo h5 {
+    font-size: 14px;
+    margin-top: 15px;
+    margin-top: 30px;
+  }
+
+  .grantInfo h3 {
+    margin-top: -20px;
+  }
+
+  .updatesButton {
+    padding-left: 30px;
+    padding-right: 30px;
+    text-decoration: none;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    background-color: #0D0764;
+    color: white;
+    border-radius: 2.7px;
+    font-size: 17px;
+  }
+
+  #grow-oss {
+    margin-top: 70px;
+    margin-bottom: 40px;
+  }
+
+  .progress {
+    width: 60%;
+    margin-left: 20%;
+    margin-top: 50px;
+    padding-bottom: 75px;
+  }
+
+  .progress > h3 {
+    font-size: 26px;
+    color: #5B3AF0;
+  }
+
+  .progressBar {
+    height: 15px;
+    background-color: #D8D8D8;
+    border-radius: 2.7px;
+  }
+
+  .progressCompleted {
+    height: 100%;
+    background-color: #0FCE7C;
+    border-radius: 2.7px;
+    float: left;
+  }
+
+  .progressRemoved {
+    height: 100%;
+    background-color: #CF0E0E;
+    border-top-right-radius: 2.7px;
+    border-bottom-right-radius: 2.7px;
+    float: left;
+  }
+
+  .funding {
+    float: right;
+    text-align: right;
+  }
+
+  .funding > h4 {
+    line-height: 0px;
+    text-align: right;
+    color: #0D0764;
+    padding-top: -20px;
+  }
+
+  .progressBarLine {
+    width: 1px;
+    height: 45px;
+    background-color: #979797;
+    float: left;
+    transform: translateY(-20%);
+    margin-right: 20px;
+  }
+
+  .funding > p {
+    line-height: 0px;
+    color: #BBBDBF;
+  }
+
+</style>
+<img id="newSupporter" src="{% static "v2/images/subscription-terminated.png" %}" alt="{% trans "Subscription Terminated." %}" title="{% trans "Subscription Terminated." %}">
+<h1 style="text-transform: none;">{% trans "Grant" %} '{{ grant.title }}' {% trans "has been cancelled by its creator." %}</h1>
+<p>{% trans "Your subscription for the project has been cancelled and you will no longer be charged." %}</p>
+<br>
+
+<hr>
+<div class="grantInfo">
+  <a href="{% url 'grants:details' grant.id grant.slug %}">
+    <img src="{{ grant.logo.url }}" style="width:50px; height: 50px;"/>
+  </a>
+  <br>
+  <a class="grantInfoName" href="{% url 'grants:details' grant.id grant.slug %}">{{ grant.title }}</a>
+  <p>{{ grant.description }}</p>
+  <div class="progress">
+    <!-- #TODO - Revisit which object to use to pull grant status. -->
```

Comment:
can this be done now?  we should have the status off the DB ojects

Reply:
Yes, I think this comment has been addressed and is a holdover from Anish's original contribution.